molecule/yaml_config_validation/converge.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Converge
+  hosts: all
+  vars_files:
+    - vars.yml
+  collections:
+    - middleware_automation.wildfly
+  roles:
+    - wildfly_install
+    - wildfly_systemd
+    - wildfly_driver
+    - wildfly_utils
+  tasks:
+    - name: Copying the setup file
```

Comment:
@RanabirChakraborty just a gut feeling, but I think you should do this either in `pre_tasks:` (so before the role is executed) or in `prepare.yml`.

Reply:
Because the `wildfly_install` role is not executed first that's why moving it to pre-tasks or in prepare.yml would through an error.

Reply:
@rpelisse deleted this task as it's not required we can deploy the configuration file directly via [this task](https://github.com/ansible-middleware/wildfly/blob/main/roles/wildfly_systemd/tasks/yml_config.yml#L45-L56)